runtime/common/src/precompile/multicurrency.rs

```diff
@@ -98,11 +98,8 @@ where
 
 		match action {
 			Action::QueryName => {
-				let name = Runtime::Erc20InfoMapping::name(currency_id).ok_or_else(|| PrecompileFailure::Revert {
-					exit_status: ExitRevert::Reverted,
-					output: "Get name failed".into(),
-					cost: target_gas_limit(target_gas).unwrap_or_default(),
-				})?;
+				// If get name failed, return ''
+				let name = Runtime::Erc20InfoMapping::name(currency_id).unwrap_or_default();
```

Comment:
I think those one can revert. Similar to querying an invalid ERC20

Reply:
Return `0x` is the default behavior

Reply:
```
{
    "jsonrpc": "2.0",
    "method": "eth_call",
    "params": [
        {
            "to": "0xdac17f958d2ee523a2206206994597c13d831ec7",
            "data": "0x06fdde03"
        },
        "latest"
    ],
    "id": 1
}
```

on ethereum, this call return the name of USDT(ERC20).
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "result": "0x0000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000a5465746865722055534400000000000000000000000000000000000000000000"
}
```

Reply:
change the `to` address to others non-erc20 address, it'll return `0x`

Reply:
https://etherscan.io/tx/0x78810f6185dec0029243d49808e657b66b034d68a493b1edd659158a19479136

FUNCTIONHASHES : `"06fdde03": "name()",`
```
{
    "jsonrpc": "2.0",
    "method": "eth_call",
    "params": [
        {
            "to": "0x881d40237659c251811cec9c364ef91dc08d300c",
            "data": "0x06fdde03"
        },
        "latest"
    ],
    "id": 1
}
```
return:
```
{
    "jsonrpc": "2.0",
    "id": 1,
    "error": {
        "code": -32000,
        "message": "execution reverted"
    }
}
```

Reply:
If the contract does not exist, return 0x, if it is a non-erc20 contract, return revert. I'll revert it.